### ISSUE TITLE
The State of Ethereum (Meetups) in Vienna in 2023 - Let's Meetup, Join Us -  Request For Comments

### ISSUE THREAD

**Issue by geraldb** [viennacrypto/viennacrypto.github.io#2]
Hello,

as posted in public to the DLT Meetup Austria:

The State of Ethereum (Meetups) in Vienna? -  here's an idea - I miss the ethereum (and tech-focused talks & chats) - see - https://www.meetup.com/Ethereum-Vienna/ - for the dead meetup page. Let's restart a new ethereum vienna group - maybe as part of DLT Austria here? - I volunteer to help to get it restarted. If anyone is interested like / comment here or talk to me in person at the meetups (I'm the "punk" pixel art fanatic). Cheers.

**Comment by davidbrai** [comment]
hey, let me know when the next meetup in vienna is. looking forward!

**Comment by geraldb** [comment]
The plan is to get the word out and meetup in a coffee house in Vienna in March 2023.  If you have an idea / suggestion for a prefered coffee house (or other venue) in Vienna, please tell.   Cheers. Prosit 2023!

**Comment by berndartmueller** [comment]
Is there any news if this is going to happen? Would be very interested to meet up

**Comment by geraldb** [comment]
@berndartmueller    Yes, it is going to happen.   But no official or any response from the old group and @ralph-pichler 

The current plan is a (social) meetup in March or April in the @ Delirium Café Vienna, Passauer Platz 2, 1. District (Beside the Church - Maria am Gestade). If you have other ideas or plan, please tell.

Otherwise, if you want to meetup in-person with my humble self - see <https://geraldb.github.io/> for some upcoming public (free) talks. / meetups.
